ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/decoders/PartialDecoderTest.kt

```diff
@@ -1,33 +1,26 @@
 package com.akuleshov7.ktoml.decoders
 
 import com.akuleshov7.ktoml.Toml
+import io.kotest.matchers.shouldBe
 import kotlinx.serialization.ExperimentalSerializationApi
 import kotlin.test.Test
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.serializer
-import kotlin.test.assertEquals
 
 @ExperimentalSerializationApi
 class PartialDecoderTest {
-    @Serializable
-    data class TwoTomlTables(val table1: Table1, val table2: Table2)
```

Comment:
I do not see this class anymore in the tests

Reply:
It was removed because it was actually not used.

In this test you are providing a toml model which as a whole is parseable to `TwoTomlTables`. However, the thing that is being tested here is partial decoding. So you don't need to use the whole model, only the model of the part you are decoding. In other words, the `TwoTomlTables` model was used only to get the `table2` value, which seems like a boilerplate to me